apps/dashboard/test/system/jobs_app_test.rb

```diff
@@ -11,18 +11,23 @@ def setup
     stub_scontrol
   end
 
-  def setup_project(dir)
-    OodAppkit.stubs(:dataroot).returns(Pathname.new(dir))
+  def setup_project(dir, override_location = false)
+    data_root = override_location ? '/tmp' : dir
+    OodAppkit.stubs(:dataroot).returns(Pathname.new(data_root))
```

Comment:
Shouldn't this be behind an if statement? I mean if we do `override_location` then the code path shouldn't even care about `OodAppkit.dataroot`. 

```ruby
OodAppkit.stubs(:dataroot).returns(Pathname.new(dir)) if override_location
```

It's just we need to be a bit careful here because we may be stubbing out functionality we actually want to test. That is - if we always stub it then we never see if the code path in the actual code is correct.